### ISSUE TITLE
Current errors in  R-CMD-check action

### ISSUE THREAD

**Issue by williamdlees** [airr-community/airr-standards#600]
I noticed these when I merged master into the germline_doc branch just now. 

1. R-CMD-check/Check will fail on the following undocumented code objects:
```
> checking for missing documentation entries ... WARNING
  Undocumented code objects:    'AlignmentSchema' 'RearrangementSchema' 'RepertoireSchema'
    'load_schema' 'read_airr_json' 'read_airr_tsv' 'read_airr_yaml'
    'read_alignment' 'read_rearrangement' 'read_repertoire'
    'validate_rearrangement' 'validate_repertoire' 'write_airr_json'
    'write_airr_tsv' 'write_airr_yaml' 'write_alignment'
    'write_rearrangement' 'write_repertoire'
```
I can see this warning if I run the check manually on my local system.

2. When the checks are run automatically as part of the pull request pipeline, they fail before this point with the error below. I can't reproduce this locally.

```
── R CMD build ─────────────────────────────────────────────────────────────────
* checking for file ‘.../DESCRIPTION’ ... OK
* preparing ‘airr’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘Usage-Vignette.Rmd’ using rmarkdown
A new version of TeX Live has been released. If you need to install or update any LaTeX packages, you have to upgrade TeX Live.
tlmgr: Local TeX Live (2021) is older than remote repository (2022).
Cross release updates are only supported with
  update-tlmgr-latest(.sh/.exe) --update
See https://tug.org/texlive/upgrade.html for details.
Warning in system2("tlmgr", args, ...) :
  running command ''tlmgr' search --file --global '/amsmath.sty'' had status 1
! LaTeX Error: File `amsmath.sty' not found.
! Emergency stop.
<read *> 
Error: processing vignette 'Usage-Vignette.Rmd' failed with diagnostics:
LaTeX failed to compile /tmp/RtmpY8cEHj/Rbuild13f47faf290/airr/vignettes/Usage-Vignette.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See Usage-Vignette.log for more info.
--- failed re-building ‘Usage-Vignette.Rmd’
SUMMARY: processing the following file failed:
  ‘Usage-Vignette.Rmd’
Error: Vignette re-building failed.
Execution halted

```
My guess is that these two problems will stop the checks running successfully on any pull request for the time being.

**Comment by javh** [comment]
I think these two message are unrelated. The first is WARNING concerning the R man pages, which is basically saying that `devtools::document()` wasn't run before `devtools::build()`.  That's fine and you can ignore it, as the documentation isn't need for the tests.

The second is an error during vignette building about a missing dependency, specifically the LaTeX math style (`amsmath.sty`), which implies a recent change to the `rocker/verse:latest` docker container. I'll have to take a look at it.

**Comment by javh** [comment]
Fixed in #593. I'll push the same change to `master`.
